fearless/Modules/Purchase/PurchaseViewFactory.swift

```diff
@@ -5,15 +5,34 @@ import CommonWallet
 final class PurchaseViewFactory: PurchaseViewFactoryProtocol {
     static func createView(
         for action: PurchaseAction,
-        commandFactory: WalletCommandFactoryProtocol
+        commandFactory _: WalletCommandFactoryProtocol
```

Comment:
it looks like you can remove this parameter at all

Reply:
Wallet use old protocol method...